### ISSUE TITLE
Invalid double -1.07502e

### ISSUE THREAD

**Issue by cbarlow26** [dnfield/vector_graphics#223]
After upgrading from 1.1.7 to 1.1.9, I'm getting the following error:

```log
E/flutter (16703): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: FormatException: Invalid double
E/flutter (16703): -1.07502e
E/flutter (16703): #0      double.parse (dart:core-patch/double_patch.dart:112:28)
E/flutter (16703): #1      parseDouble (package:vector_graphics_compiler/src/svg/numbers.dart:32:17)
E/flutter (16703): #2      _parseTransformParams (package:vector_graphics_compiler/src/svg/parsers.dart:34:20)
E/flutter (16703): #3      parseTransform (package:vector_graphics_compiler/src/svg/parsers.dart:71:33)
E/flutter (16703): #4      _Elements.linearGradient (package:vector_graphics_compiler/src/svg/parser.dart:334:45)
E/flutter (16703): #5      SvgParser._parseTree (package:vector_graphics_compiler/src/svg/parser.dart:779:20)
E/flutter (16703): #6      SvgParser.parse (package:vector_graphics_compiler/src/svg/parser.dart:799:5)
E/flutter (16703): #7      parse (package:vector_graphics_compiler/vector_graphics_compiler.dart:76:17)
E/flutter (16703): #8      encodeSvg (package:vector_graphics_compiler/vector_graphics_compiler.dart:143:5)
E/flutter (16703): #9      SvgLoader._load.<anonymous closure>.<anonymous closure> (package:flutter_svg/src/loaders.dart:137:14)
E/flutter (16703): #10     _testCompute (package:flutter_svg/src/utilities/compute.dart:12:38)
E/flutter (16703): #11     SvgLoader._load.<anonymous closure> (package:flutter_svg/src/loaders.dart:135:21)
```

There appears to be a problem with parsing the `gradientTransform` attribute, because the actual value double value (-1.07502e-05) is correct. Please see the svg below:

```xml
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="100%" height="100%" viewBox="0 0 653 650" version="1.1"
	xmlns="http://www.w3.org/2000/svg"
	xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve"
	xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
	<path d="M326.338,529.156l61.418,87.513l20.475,-104.907l91.7,54.957l-23.956,-104.176l106.144,12.919l-64.269,-85.45l102.2,-31.343l-93.45,-51.919l80.606,-70.212l-106.481,-9.413l45.1,-96.95l-101.138,34.725l1.769,-106.9l-78.231,72.837l-41.887,-98.375l-41.844,98.375l-78.269,-72.837l1.75,106.9l-101.1,-34.725l45.087,96.95l-106.506,9.413l80.625,70.212l-93.45,51.919l102.219,31.343l-64.269,85.45l106.138,-12.919l-23.982,104.176l91.707,-54.957l20.512,104.907l61.382,-87.513Z" style="fill:#fee849;fill-rule:nonzero;"/>
	<path d="M326.338,97.243l68.243,-97.243l22.769,116.587l101.875,-61.081l-26.619,115.744l117.938,-14.35l-71.413,94.944l113.55,34.843l-103.812,57.663l89.544,78.013l-118.294,10.468l50.081,107.713l-112.375,-38.569l1.987,118.769l-86.962,-80.932l-46.512,109.3l-46.507,-109.3l-86.95,80.932l1.956,-118.769l-112.343,38.569l50.087,-107.713l-118.325,-10.468l89.594,-78.013l-103.85,-57.663l113.587,-34.843l-71.425,-94.944l117.932,14.35l-26.632,-115.744l101.888,61.081l22.781,-116.587l68.207,97.243Z" style="fill:#feb81b;fill-rule:nonzero;"/>
	<path d="M542.056,324.562c0,119.125 -96.6,215.707 -215.718,215.707c-119.132,-0 -215.694,-96.582 -215.694,-215.707c-0,-119.137 96.562,-215.7 215.694,-215.7c119.118,0 215.718,96.563 215.718,215.7Z" style="fill:#fee849;fill-rule:nonzero;"/>
	<path d="M516.594,324.562c-0,105.069 -85.182,190.25 -190.256,190.25c-105.069,0 -190.244,-85.181 -190.244,-190.25c-0,-105.081 85.175,-190.25 190.244,-190.25c105.074,0 190.256,85.169 190.256,190.25Z" style="fill:#f7db69;fill-rule:nonzero;"/>
	<path d="M326.338,142.768c-100.238,0 -181.788,81.557 -181.788,181.794c0,100.238 81.55,181.788 181.788,181.788c100.243,-0 181.793,-81.55 181.793,-181.788c0,-100.237 -81.55,-181.794 -181.793,-181.794Zm-0,380.507c-109.563,-0 -198.707,-89.144 -198.707,-198.713c0,-109.568 89.144,-198.712 198.707,-198.712c109.568,-0 198.718,89.144 198.718,198.712c0,109.569 -89.15,198.713 -198.718,198.713Z" style="fill:#feb81b;fill-rule:nonzero;"/>
	<path d="M325.412,418.837c-43.618,0 -85.868,-19.993 -114.837,-54.368c-31,-36.757 -39.306,-85.894 -14.031,-127.957c27.493,-45.793 77.712,-73.756 128.868,-73.756c50.244,0 98.144,26.594 126.613,70.163c30.231,46.268 16.819,103.481 -20.406,141.025c-28.425,28.662 -66.819,44.893 -106.207,44.893Z" style="fill:url(#_Linear1);"/>
	<defs>
		<linearGradient id="_Linear1" x1="0" y1="0" x2="1" y2="0" gradientUnits="userSpaceOnUse" gradientTransform="matrix(-1.07502e-05,245.936,-245.936,-1.07502e-05,325.153,172.034)">
			<stop offset="0" style="stop-color:#fdf9e5;stop-opacity:1"/>
			<stop offset="1" style="stop-color:#f7db69;stop-opacity:1"/>
		</linearGradient>
	</defs>
</svg>
```

Downgrading to 1.1.7 solves the issue.

**Comment by dnfield** [comment]
This is actually a bug in vector_graphics. I thought it was fixed but, alas, it was not. Looking now.

**Comment by dnfield** [comment]
Something went wrong with publishing vector_graphics 1.0.9. I've published 1.0.9+1  which shoudl fix this. Please run `dart pub upgrade`.
